astropy/table/table.py

```diff
@@ -494,6 +494,15 @@ def loc(self):
         return TableLoc(self)
 
     @property
+    def loc_indices(self):
+        '''
+        Return a TableLocIndices object that can be used for retrieving
+        rows by index in a given data range. Note that both loc
+        and iloc work only with single-column indices.
+        '''
```

Comment:
Put an example like this in the doc string and in the narrative docs:
```
>>> t = Table([('apple', 'peach', 'pear', 'banana'), (10, 1, 9, 9)], names=('name', 'number'))
>>> t.add_index('name')
>>> t.loc_indices['peach']
1
>>> t.loc_indices[['banana', 'peach']]
[3, 1]
```

Reply:
narrative docs?

Reply:
E.g. like this: http://astropy.readthedocs.io/en/latest/table/indexing.html#row-retrieval-using-indices